### PR TITLE
improve check mark quality 

### PR DESCRIPTION
**Pull request by kamalika0363**
1. Changed the Check image (added .png file) and reduced height and width in JS File. 
2. README file's Screenshot Changed to a CID link.

image

FMC SS

### CODE REVIEW

**Review by TejasNair9977 — APPROVED** [review]
looks good to me!

**Review by TejasNair9977 — APPROVED** [review]
LGTM

File: screenshot.jpg
Comment (demonkillerr) [inline screenshot.jpg:1]: i think we need an updated screenshot. would be nice if you could do that

**Review by demonkillerr — APPROVED** [review]

**Comment by demonkillerr** [comment]
approved and merged

File: screenshot.jpg
Comment (kamalika0363) [reply screenshot.jpg:1]: Do we also need that play store image and owner info too?

Comment (demonkillerr) [reply screenshot.jpg:1]: i don't think so... it would be better to add contributors separately in the readme file

**Pull request merged**